lib/components/TinyMCE.js

```diff
@@ -51,12 +52,19 @@ const TinyMCE = React.createClass({
   },
 
   componentDidMount() {
-    const config = clone(this.props.config);
-    this._init(config);
+    this._init(this.props.config, this.props.content);
   },
 
   componentWillReceiveProps(nextProps) {
-    if (!isEqual(this.props.config, nextProps.config)) {
+    if (!isEqual(this.props.config, nextProps.config, function(c1, c2) {
+		    if(typeof c1 === 'function' && typeof c2 === 'function') {
+			    return c1.toString() === c2.toString();
+		    }
+		    if(c1.hasOwnProperty('items') && c2.hasOwnProperty('items')) {
```

Comment:
the check for items is possibly unnecessary as the config is cloned before given to tinymce so the items properties are not modified anymore